web/modules/custom/joinup_user/joinup_user.install

```diff
@@ -6,66 +6,12 @@
  */
 
 use Drupal\Core\File\FileSystemInterface;
-use Drupal\Core\Session\AccountInterface;
 use Drupal\file\Entity\File;
 
 /**
  * Implements hook_install().
  */
 function joinup_user_install() {
-  $authenticated_permissions = [
-    'access comments',
-    'access content',
-    'access contextual links',
-    'access tour',
-    'access user profiles',
-    'cancel account',
-    'create contact_information rdf entity',
-    'create owner rdf entity',
-    'create document content',
-    'delete own comments',
-    'delete own document content',
-    'edit own comments',
-    'edit own contact_information rdf entity',
-    'edit own owner rdf entity',
-    'edit own tallinn_report content',
-    'export rdf metadata',
-    'flag subscribe_discussions',
-    'never autoplay videos',
-    'post comments',
-    'propose collection rdf entity',
-    'skip comment approval',
-    'unflag subscribe_discussions',
-    'update own document content',
-    'use text format basic_html',
-    'use text format content_editor',
-    'use text format simple_html',
-    'view own unpublished content',
-    'view published og menu instance entities',
-    'view rdf entity',
-  ];
-  user_role_grant_permissions(AccountInterface::AUTHENTICATED_ROLE, $authenticated_permissions);
-
-  $anonymous_permissions = [
-    'access comments',
-    'access content',
-    'access tour',
-    'access user profiles',
-    'use text format basic_html',
-    'use text format simple_text',
-    'view rdf entity',
-    'export rdf metadata',
-    'access comments',
-  ];
-  user_role_grant_permissions(AccountInterface::ANONYMOUS_ROLE, $anonymous_permissions);
-
-  joinup_user_setup_default_avatar();
-}
-
-/**
- * Set a default image for the users avatar.
- */
-function joinup_user_setup_default_avatar() {
```

Comment:
Can we keep this function as is used in `joinup_user_post_update_reset_default_icons()`? I know that the post update already ran on PROD but is not necessary true for a 3rd party using our platform.